fip-0026.md

```diff
@@ -238,85 +269,73 @@ This action will collect the FIO from the buyer, transfer the domain from holder
 |Insufficient Funds|`buyer` doesn't have enough funds|400| | |"Not enough FIO"|
 |Actor not signer|The signer and the `actor` do not match |403| | |Type: invalid_signature|
 
-#### Response
-TBD
-
-##### Example
-TBD
-
-### Renew Listing
-
-#### New action: *rnlistdomain*
-This action can be called by an actor that already has a domain listed for sale and it's within the `warn_time` window. This is the fee-less way to keep the domain for sale as long as the domain is not going to expire before the next expiration date. 
-
-#### Request
-|Parameter|Required|Format|Definition|
-|---|---|---|---|
-|actor|Yes|12 character string|Valid actor of signer|
-|fio_domain|Yes|FIO Domain|Valid FIO Domain that is already listed|
+#### Response body
+|Parameter|Format|Definition|
+|---|---|---|
+|status|String|OK if successful|
+|fee_collected|Int|Amount of SUFs collected as fee|
 
 ##### Example
 ```
 {
-    "actor": "2odzomo2v4pe",
-    "fio_domain": "alice"
+  "status": "OK",
+  "fee_collected": 2000000000
 }
 ```
 
-#### Processing
-
-- Request is validated per exception handling
-- Calculate expiration date by taking current expiration date and adding on `duration` time. 
-- Write new expiration date to table
-
-#### Exception handling
-|Error condition|Trigger|Type|fields:name|fields:value|Error message|
-|---|---|---|---|---|---|
-|Invalid FIO domain|Supplied `fio_domain` is not listed for sale.|400| | |"FIO domain not found"|
-|Domain will expire within `duration`|Domain will expire before new listing expiration date|400| | |"Domain will expire before next expiration of listing"|
-|Outside `warn_time` window|The listing does not expire within `warn_time` days.|400| | |"Too early to renew listing"|
-|Actor not signer|The signer and the `actor` do not match |403| | |Type: invalid_signature|
-
-#### Response
-TBD
-
-##### Example
-TBD
-
 ### Set Holder Account
```

Comment:
This action was deleted in this PR. It was to renew listings without incurring a new listing fee. It was never written and removed because it was decided we should not have a 90 day limit on the listing per discussion with Luke.